functional-test/src/test/groovy/org/rundeck/util/common/jobs/JobUtils.groovy

```diff
@@ -358,59 +377,60 @@ class JobUtils {
         tempFile.deleteOnExit()
         tempFile.path
     }
-    static void validateJobsImportAllSuccess(Map data){
-        if(data.failed && data.failed.size()>0){
-            throw new Exception("Some jobs failed on import: "+data.failed)
+
+    /**
+     * Executes the handler if at least one job import fails
+     * @param data
+     * @param failedJobsHandler
+     */
+    private static void validateJobsImportAllSuccess(data,
+                                                     Consumer<CreateJobResponse> failedJobsHandler = { List<Object> failedJobs -> throw new Exception("Some jobs failed on import: " + failedJobs) } ) {
+        if(data?.failed?.size()>0) {
+            failedJobsHandler.accept(data?.failed)
```

Comment:
maybe a cleaner way is to use an override with one parameter, and pass the default handler to 2nd method definition